### ISSUE TITLE
Training MultiHMR + Anny

### ISSUE THREAD

**Issue by leonardwetzel** [naver/multi-hmr#65]
Hi naver team, thanks for your impressive work regarding Anny! I am trying to recreate the training process of the MultiHMR + Anny checkpoint and have two questions:

1.) What dataset does "ITW" refer to exactly in the pre-trained model table?
2.) How do you train on BEDLAM using the Anny model? Do you use the Anny SMPL-X forward and compute the loss against the ground truth SMPL-X topology mesh?

Thanks in advance!
